test/e2e/metrics_util.go

```diff
@@ -161,8 +175,8 @@ func HighLatencyRequests(c *client.Client, threshold time.Duration) (int, error)
 }
 
 // Verifies whether 50, 90 and 99th percentiles of PodStartupLatency are smaller
-// than the given threshold (returns error in the oposite case).
-func VerifyPodStartupLatency(latency PodStartupLatency, podStartupThreshold time.Duration) error {
+// than the 5 seconds threshold (returns error in the oposite case).
```

Comment:
I'm not sure I would list specific numbers here, since you'll need to update it every time the constant changes.